co-authors-plus.php

```diff
@@ -1833,6 +1838,39 @@ public function filter_pre_get_avatar_data_url( $args, $id ) {
 		}
 		return $args;
 	}
+
+	/**
+	 * Conditionally Hide Author Term Description
+	 * 
+	 * If the current user does not have the required capability,
+	 * hide the author term description by unsetting it.
+	 *
+	 * @link https://github.com/Automattic/Co-Authors-Plus/issues/930
+	 * @param WP_REST_Response $response Response for an individual author taxonomy term.
+	 * @return WP_REST_Response $response Same response, possibly mutated to eliminate value of description.
+	 */
+	public function conditionally_hide_author_term_description( WP_REST_Response $response ) : WP_REST_Response {
+		$capability = apply_filters(
+			'coauthors_view_author_description_capability',
```

Comment:
Just a nitpick, for the naming, can we call it `coauthors_rest_view_author_description_cap`?

Reply:
Yes, this is updated.